indicators/NO_BIRD_001-004/R/NO_BIRD_001-004.qmd

```diff
@@ -0,0 +1,624 @@
+
+```{r}
+#| code-summary: "Create local directories"
+## Absolute path for root folder:
+wd <- here::here("indicators/NO_BIRD_001-004")
+
+# Storage of raw PECBMS Trim outputs
+folder <- paste0(wd, "/PECBMS_Files")
+
+# Storage of processed PECBMS Trim outputs
+subFolderName <- "Species_files"
+
+# general and working folders
+ general_folder_rel <- paste0(wd, "/temp/birdIndicators-main/data")
+ working_folder_rel <- paste0(folder, "/", subFolderName)
+
+# Storage of MSI results
+MSI_results_folder <- paste0(wd, "/MSI_Results")
+
+# NINAs local storage of legacy Trim output files
+legacyFile_folder <- "P:/41201612_naturindeks_2021_2023_database_og_innsynslosning/Hekkefugl_Dataflyt/LegacyFiles_PECBMS_Trim"
+
+# Absolute data and output paths for RSWAN (not needed)
+# general_folder <- general_folder_rel
+# working_folder <- working_folder_rel
+# output_folder <- paste0(working_folder, "/output/") 
+# output_folder2 <- paste0(working_folder, "/output") 
+
+
+## Create directories if they are not available
+dir.list <- c(
+  folder, 
+  general_folder_rel, 
+  working_folder_rel
+  )
+
+for(i in 1:length(dir.list)){
+  if(!file.exists(dir.list[i])){ 
+  dir.create(dir.list[i])
+  }
+}
+
+```
+
+### Data sets
+
+The workflow uses the up-to-date raw data from the Norwegian bird monitoring. One can also combine this with the calculated abundance indices from the predecessor monitoring scheme in Norway, and that way get a longer time series, but we will not do that here. Additionally, we also use a couple of lists holding information on which species (and years) are to be included in which parts of the analytical workflow as the different outcomes require different information.
+
+### Monitoring data {#sec-download}
+
+The route-based raw data from “hekkefuglovervåking” are compiled after each breeding season and curated and stored in NINA’s database. Before storing new data in the database, a thorough quality-check is performed. From that database we download the data relevant for the [PECBMS Trim analyses](https://pecbms.info/methods/software/trim/) and for the specified range of years (here from 2008 up to and including 2024). The function `downloadData_TRIM` does that for a specified year range (from minYear up to and including maxYear), and requires specifying which driver is used for accessing the database (via argument DriverName). When running from the NINA RStudio server, we need to use “FreeTDS”. When running locally from NINA computers, we need to select “SQL Server”. Note that downloading these data requires user-specific access privileges which can be requested from NINA IT department (‘datahjelp’).
+
+```{r}
+#| eval: false
+#| code-summary: "Download Trim data from NINA database"
+minYear <- 2008
+maxYear <- 2024
+
```

Comment:
I don't have this function

Reply:
Get it from line 246. But you also dont have acess to the database, and that's why the code is `eval: false`